### ISSUE TITLE
There are many differences in accuracy. 

### ISSUE THREAD

**Issue by modernkim** [NVIDIA-AI-IOT/torch2trt#236]
There is a difference between FP32 and tensorrt FP32.
The data set is CIFAR100. Am I doing the wrong conversion?
~~~
def TensorRT(epoch):
    # make_sample_image
    sample_img = np.random.rand(args.trtbatch,args.trtchannels,args.trtpixels,args.trtpixels)
    if args.trtFP == 32:
        # dummy_input FP 32
        input = torch.from_numpy(sample_img).view(args.trtbatch,args.trtchannels,args.trtpixels,args.trtpixels).float().to(device)
        tensorrt_net = args.network
        tensorrt_net.load_state_dict(torch.load(args.weight_path))
        tensorrt_net.float().to(device)
        # convert trt model FP 32
        net_trt = torch2trt(tensorrt_net,[input],fp16_mode = False,max_batch_size = args.trtbatch)
        net_trt.to(device)

    elif args.trtFP == 16:
        # dummy_input FP 16
        input =  torch.from_numpy(sample_img).view(args.trtbatch,args.trtchannels,args.trtpixels,args.trtpixels).half().to(device)
        tensorrt_net = args.network
        tensorrt_net.load_state_dict(torch.load(args.weight_path))
        tensorrt_net.half().to(device)
        # convert trt model FP 16
        net_trt = torch2trt(tensorrt_net,[input],fp16_mode = True,max_batch_size=args.trtbatch)
        net_trt.to(device)

    elif args.trtFP == 8:
        # dummy_input INT 8
        input = torch.from_numpy(sample_img).view(args.trtbatch,args.trtchannels,args.trtpixels,args.trtpixels).char().to(device)
        tensorrt_net = args.network
        tensorrt_net.load_state_dict(torch.load(args.weight_path))
        tensorrt_net.to(device)
        #convert trt model INT 8
        net_trt = torch2trt(tensorrt_net,[input],fp16_mode = False,int8_mode = True,max_batch_size = args.trtbatch)
        net_trt.to(device)

    print(input.dtype)
    print('=========Start_TensorRT_Check==========')
    net_trt.eval()
    correct = 0
    total = 0
    avg_frame_total = 0.
    TensorRT_time = []
    with torch.no_grad():
            for batch_idx, (inputs, targets) in enumerate(testloader):
                # convert inputs dtype
                if args.trtFP == 32:
                        inputs = Variable(inputs).to(device)
                elif args.trtFP == 16:
                        inputs = Variable(inputs).half().to(device)
                elif args.trtFP == 8:
                        inputs = Variable(inputs).char().to(device)
                targets = Variable(targets).to(device)
            tic = time.time()
            outputs = net_trt(inputs)
            toc = time.time()

            if batch_idx != 0:
                TensorRT_time.append(toc-tic)
                avg_frame_total += (TensorRT_time[-1])
                avg_frame = avg_frame_total/batch_idx
                avg_frame = 1/avg_frame
            _, predicted = outputs.max(1)
            total += targets.size(0)
            correct += predicted.eq(targets).sum().item()

            if ((batch_idx != 0)and(batch_idx%2000==0))or batch_idx==9999:
                print(batch_idx, len(testloader), 'Acc: %.4f%% (%d/%d),TRT Speed is : (%.3f [FPS])'
                % (100.*correct/total, correct, total, avg_frame))

for epoch in range(1):
    FP32(epoch)
    FP16(epoch)
    TensorRT(epoch)
~~~

**Comment by zwqnju** [comment]
Maybe your issue is similar with https://github.com/NVIDIA-AI-IOT/torch2trt/issues/220

I also met an accuracy problem, and fixes it with x = x.contiguous()

**Comment by modernkim** [comment]
@zwqnju I'll try it ~ thanks

**Comment by modernkim** [comment]
Good~
